### PR TITLE
add rust readme files

### PR DESCRIPTION
**Pull request by billyoung**

### CODE REVIEW

**Review by autodidaddict — CHANGES_REQUESTED** [review]
in the readmes, you can remove the code lines that begin with "# ", e.g. 
```
# fn start_download(req: StreamRequest) -> HandlerResult<()> {
#   Ok(())
# }
``` 
This means it's hidden from the generated rust documentation, so it can be deleted from the readme as well.

File: extras/rust/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  let result = json!({"guid": guid });
+  Ok(http::Response::json(&result, 200, "OK"))
+//!
+}
+//!
+fn health(_: core::HealthCheckRequest) -> HandlerResult<core::HealthCheckResponse> {
+  Ok(core::HealthCheckResponse::healthy())   
+}
+//!
+# fn get_guid() -> HandlerResult<Option<String>> {
```
Comment (autodidaddict) [inline extras/rust/README.md:45]: this function can be deleted from the readme.

File: graphdb/rust/README.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    });
+
+    Ok(http::Response::json(result, 200, "OK"))
+}
+
+fn health(_: actor_core::HealthCheckRequest) -> HandlerResult<actor_core::HealthCheckResponse> {
+  Ok(actor_core::HealthCheckResponse::healthy())   
+}
+
+# fn query_db<T: graph::FromTable>(graph_name: String, query: String) -> ::wapc_guest::HandlerResult<T> {
```
Comment (autodidaddict) [inline graphdb/rust/README.md:46]: nuke this from orbit.

File: http-client/rust/README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            status: res.status,
+            header: res.header,
+            body: res.body,
+        })
+    } else {
+        Ok(httpserver::Response::internal_server_error("Only GET requests can be proxied with this actor"))
+    }
+}
+
+# fn request(method: String, url: String, headers: std::collections::HashMap<String,String>, body: Vec<u8>) -> HandlerResult<httpclient::Response> {
```
Comment (autodidaddict) [inline http-client/rust/README.md:47]: delete this.

**Review by autodidaddict — CHANGES_REQUESTED** [review]
just a couple things to remove from the readmes.

File: eventstreams/rust/README.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+extern crate wasmcloud_actor_eventstreams as streams;
+// extern crate actor_core as actorcore;
+use wapc_guest::HandlerResult;
+use streams::StreamQuery;
+use std::collections::HashMap;
+
+#[no_mangle]
+pub fn wapc_init() {
+    streams::Handlers::register_deliver_event(deliver_event);
+//     actorcore::Handlers::register_health_request(health);
```
Comment (brooksmtownsend) [inline eventstreams/rust/README.md:18]: This `extern crate` can be removed as long as we uncomment the wapc_init and rename it to `wasmcloud_actor_core`
Also let's throw in the following function below `deliver_event` for acurracy
```rust
fn health(_h: core::HealthCheckRequest) -> HandlerResult<core::HealthCheckResponse> {
    Ok(core::HealthCheckResponse::healthy())
}
```

File: extras/rust/README.md (hunk 1 above)
Comment (brooksmtownsend) [inline extras/rust/README.md:45]: The sparse `//!`s can be dropped

File: graphdb/rust/README.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+    let result = json!({
+        "name": name,
+        "birth_year": birth_year
+    });
+
+    Ok(http::Response::json(result, 200, "OK"))
+}
+
+fn health(_: actor_core::HealthCheckRequest) -> HandlerResult<actor_core::HealthCheckResponse> {
+  Ok(actor_core::HealthCheckResponse::healthy())
```
Comment (brooksmtownsend) [inline graphdb/rust/README.md:43]: references to `actor_core` here should be `wasmcloud_actor_core`

File: graphdb/rust/README.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+allows actors to interact with a graph database, such as RedisGraph or Neo4j.
+
+Example:
+
+```rust
+use serde_json::json;
+extern crate wapc_guest as guest;
+use actor_graphdb as graph;
+use graph::*;
+use actor_http_server as http;
```
Comment (brooksmtownsend) [inline graphdb/rust/README.md:13]: These two should have `wasmcloud` prefix

File: telnet/rust/README.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+```rust
+extern crate wasmcloud_actor_telnet as telnet;
+// extern crate wasmcloud_actor_core as actorcore;
+use wapc_guest::HandlerResult;
+
+#[no_mangle]
+pub fn wapc_init() {
+    telnet::Handlers::register_session_started(session_started);
+    telnet::Handlers::register_receive_text(receive_text);
+//     actorcore::Handlers::register_health_request(health);
```
Comment (brooksmtownsend) [inline telnet/rust/README.md:18]: You can uncomment these just fine as long as you throw in a health check function:
```rust

fn health(_h: core::HealthCheckRequest) -> HandlerResult<core::HealthCheckResponse> {
    Ok(core::HealthCheckResponse::healthy())
}

```

**Review by brooksmtownsend — CHANGES_REQUESTED** [review]
Just a few changes, mostly `wasmcloud_` prefix related

**Comment by billyoung** [comment]
thanks for the feedback! @brooksmtownsend @autodidaddict should be resolved now

File: blobstore/rust/README.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+    let verse = String::from_utf8(chunk.chunk_bytes)?;
+    info!("Poem {} part {}:\n{}", chunk.id, chunk.sequence_no, verse);
+    Ok(())
+}
+
+fn health(_: actor_core::HealthCheckRequest) -> HandlerResult<actor_core::HealthCheckResponse> {
+  Ok(actor_core::HealthCheckResponse::healthy())   
+}
+
+# fn start_download(req: StreamRequest) -> HandlerResult<()> {
```
Comment (autodidaddict) [inline blobstore/rust/README.md:56]: This function should still be taken out of the readme.

**Review by autodidaddict — CHANGES_REQUESTED** [review]

File: blobstore/rust/README.md (hunk 8 above)
Comment (billyoung) [reply blobstore/rust/README.md]: thanks! not sure how i missed that. fixed.

**Review by autodidaddict — APPROVED** [review]

File: eventstreams/rust/README.md [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+```rust
+extern crate wasmcloud_actor_eventstreams as streams;
+use wapc_guest::HandlerResult;
+use streams::StreamQuery;
+use std::collections::HashMap;
+
+#[no_mangle]
+pub fn wapc_init() {
+    streams::Handlers::register_deliver_event(deliver_event);
+    wasmcloud_actor)core::Handlers::register_health_request(health);
```
Comment (brooksmtownsend) [inline eventstreams/rust/README.md:17]: Heh, all good except for this little typo 😄

**Review by brooksmtownsend — CHANGES_REQUESTED** [review]

File: eventstreams/rust/README.md (hunk 9 above)
Comment (brooksmtownsend) [reply eventstreams/rust/README.md:17]: `)`

**Pull request merged**